core-common/src/main/java/org/zalando/nakadi/repository/kafka/KafkaTopicRepository.java

```diff
@@ -299,41 +300,55 @@ public boolean topicExists(final String topic) throws TopicRepositoryException {
     public void syncPostBatch(
             final String topicId, final List<BatchItem> batch, final String eventType, final boolean delete)
             throws EventPublishingException {
+
         final Producer<byte[], byte[]> producer = kafkaFactory.takeProducer();
         try {
-            final Map<String, String> partitionToBroker = producer.partitionsFor(topicId).stream()
```

Comment:
it is not clear how this is related to the change

Reply:
It's becomes harder to recall what was the reason to change this part, but it looks like in the context of the bigger change we don't have to track this anymore.  Esp. since we intend to enable retries, the item may be sent to a different broker in a repeated attempt, so it's no longer a 1:1 relation.

Reply:
I am not against removing it, just maybe a separate PR that it does not affect actual change